tests/integration/components/frost-bunsen-detail/conditional-views-test.js

```diff
@@ -0,0 +1,81 @@
+import {expect} from 'chai'
+import {setupComponentTest} from 'ember-mocha'
+import wait from 'ember-test-helpers/wait'
+import hbs from 'htmlbars-inline-precompile'
+import {describe, it, beforeEach} from 'mocha'
+
+function renderWithValue (context, value) {
+  context.set('value', value)
+  context.render(hbs`{{frost-bunsen-form
+    bunsenModel=bunsenModel
+    bunsenView=bunsenView
+    hook=hook
+    value=value
+  }}`)
+}
+
+describe('Integration: frost-bunsen-form / views with conditions', function () {
+  setupComponentTest('forms with conditions', {
```

Comment:
It is easier than this https://github.com/ciena-frost/ember-frost-bunsen/blob/63504aaec3ece532be36defd58069d83e393c358/tests/integration/components/frost-bunsen-form/single-root-container-test.js#L4